### DISCUSSION TITLE
Question regarding FAQ

### DISCUSSION THREAD

**Discussion by scrog** [schemaorg/schemaorg#3479] [Q&A]
On our site we have a lot of question and answers (advice regarding food).
The question/answers is accesible thru a search driven page and not a FAQ page.

We are thinking about creating a FAQ-page for each added food question/answer and structured data (FAQ-type) for each page.
Would that be considered an correct solution or would it be better to try and group similar food on a FAQ-page with several food/answers and several FAQ-type objects on the same page?

Our goal is to get better visibility on searchpages for relevant searches.

**Comment by MatthiasWiesmann** [comment]
This is an SEO question, not a schema.org question.
